### DISCUSSION TITLE
VS Code Autocomplete doesn't suggest status codes

### DISCUSSION THREAD

**Discussion by assman** [fastapi/fastapi#8880] [Questions]
Trying to use the status codes but my editor doesn't show any suggestions.

![Screen Shot 2021-05-29 at 9 38 08 PM](https://user-images.githubusercontent.com/14093665/120072447-91f8ee80-c0c6-11eb-8de2-68f40b0ae6bc.png)

**Comment by Kludex** [comment]
Well... You'll not see a suggestion after `created`... You can see `HTTP_201_CREATED` there...

**Comment by assman** [comment]
So there's only one convenience variable in there? What about other status codes?

**Comment by Kludex** [comment]
If you write `status.HTTP`, can't you see all of them?

**Comment by assman** [comment]
Nope - check this out

![ezgif-2-4f111ec6d49e](https://user-images.githubusercontent.com/14093665/120073273-3cbedc00-c0ca-11eb-883e-8f485351f686.gif)

**Comment by KiraPC** [comment]
@assman 

It seems working on my configuration.

<img width="891" alt="Schermata 2021-05-30 alle 21 34 11" src="https://user-images.githubusercontent.com/46629148/120117762-06976000-c18f-11eb-96ff-b84c2206b0f9.png">

So I think is not a FastAPI issue.

**Comment by assman** [comment]
> @assman 
> 
> 
> 
> It seems working on my configuration.
> 
> 
> 
> <img width="891" alt="Schermata 2021-05-30 alle 21 34 11" src="https://user-images.githubusercontent.com/46629148/120117762-06976000-c18f-11eb-96ff-b84c2206b0f9.png">
> 
> 
> 
> So I think is not a FastAPI issue.

Are you able to see all the suggestions? Your screenshot is only showing 201 right?

**Comment by KiraPC** [comment]
@assman yeh, I can see any suggestion. In this case I tape creation so it is showing just the matched one.

**Comment by nzig** [comment]
@assman What Python plugins do you have installed in VSCode?
